landscape/lib/tests/test_network.py

```diff
@@ -266,7 +282,7 @@ def test_skip_iface_with_no_ip(self, mock_interfaces, mock_ifaddresses):
     @patch("landscape.lib.network.get_flags")
     @patch("landscape.lib.network.netifaces.ifaddresses")
     @patch("landscape.lib.network.netifaces.interfaces")
-    def test_skip_iface_down(
+    def test_iface_down(
```

Comment:
Since we're already making edits to this section of the code, would you mind adding a docstring to the test to document this new behavior?  Something like "Interfaces in the 'down' state are still reported"